### ISSUE TITLE
Add coverage support

### ISSUE THREAD

**Issue by zainab-ali** [to-ithaca/sbt-directive#24]
The `scoverage` `coverageReport` task fails with the error:

```
No source root found for '/target/directive/generatedSources/$FILENAME' (source roots: '')
```

We need to add the original sources as roots that `scoverage` can understand (there's no point adding the generated sources).

**Comment by zainab-ali** [comment]
It looks like we need to put the generated sources in `sourceManaged` for this to work.

**Comment by zainab-ali** [comment]
Fixed in PR #25
